### PR TITLE
Allows token usage in manual_glidein_submit. Fixes #140

### LINKED ISSUE #140: manual_glidein_submit is not using tokens to submit pilots
**Describe the bug**
manual_glidein_submit is not setting the scitoken before calling the submitGlidein function

**To Reproduce**
Simply call the script `manual_glidein_submit --entry-name=CMSHTPC_T2_US_Caltech_cit2_gpu --fe-name=fecmsglobal` and verify if the scitoken is set `entry_q CMSHTPC_T2_US_Caltech_cit2 -af scitokensfile`

**Expected behavior**
If available, the scitoken should be set in the data structures passed as arguments to submitGlidein

### PR DESCRIPTION
**Pull request by mmascher**

### CODE REVIEW

**Comment by mambelli** [comment]
Pylint is failing, did you run pre-commit?

File: factory/tools/manual_glidein_submit.py [hunk 1]
```diff
@@ -199,6 +199,15 @@ def main():
...
             )
+        scitoken = "credential_{}_{}.scitoken".format(ad_gc['ClientName'], entry_name)
+        scitoken_file = os.path.join(credentials.cred_dir, scitoken)
+        if not os.path.exists(scitoken_file):
+            logging.warning("Cannot find scitoken file %s" % scitoken_file)
+        elif not credentials.add_identity_credential("frontend_scitoken", scitoken_file):
+            loging.warning(
+                "failed to add frontend_scitoken %s to identity credentials %s"
+                % (scitoken_file, str(credentials.identity_credentials))
+
```
Comment (mambelli) [inline factory/tools/manual_glidein_submit.py:210]: I think you should have `)` here

**Review by mambelli — CHANGES_REQUESTED** [review]

**Comment by mmascher** [comment]
> Pylint is failing, did you run pre-commit?

done now, apologies

**Review by mambelli — APPROVED** [review]

**Comment by mambelli** [comment]
Pylint was still failing. I added the fix

**Pull request merged**